test/benchmark/CMakeLists.txt

```diff
@@ -22,6 +22,18 @@ target_include_directories(bm_proto_creation PUBLIC
 
 target_link_libraries(bm_proto_creation
     benchmark
+    gtest::gtest
```

Comment:
Please remove gtest and gmock here and below, because they do not look used in the code.

Reply:
They are used in test validator, which is used in test transaction builder